ItemsListApp/test/ItemsListApp.Api.UnitTests/Tests/Controllers/ItemsControllerUnitTests.cs

```diff
@@ -123,23 +151,62 @@ public void SetUp()
         }
 
         [Test]
-        public async Task Put_ItemWithId_ReturnsSameItem()
+        public async Task Put_ItemWithId_ReturnsSuccessCodeRequest()
         {
             var expected = new Item
             {
                 Id = new Guid("999EA6F0-4139-4D54-B4DD-4976A35D1DFA"),
-                Text = "Text of required item",
+                Text = "InvalidateText of required item",
```

Comment:
Rename noise? :rofl: 
Is it copy paste error you missed this when checking changes prior commit?